src/OpenLoco/Window.cpp

```diff
@@ -1526,8 +1526,8 @@ namespace OpenLoco::Ui
                     Widget::draw_15(dpi, this, widget, widgetFlags, colour, disabled);
                     break;
 
-                case widget_type::wt_16:
-                    assert(false); // Unused
+                case widget_type::groupbox:
```

Comment:
Do we know its groupbox or is this just copying RCT2?

Reply:
Is copying OpenRCT2. I actually think Locomotion doesn't have code for groupboxes.